mm2src/mm2_main/src/lp_swap.rs

```diff
@@ -236,10 +237,12 @@ pub fn broadcast_p2p_tx_msg(ctx: &MmArc, topic: String, msg: &TransactionEnum, p
     broadcast_p2p_msg(ctx, vec![topic], encoded_msg, from);
 }
 
-pub async fn process_msg(ctx: MmArc, topic: &str, msg: &[u8]) {
+pub async fn process_msg(ctx: MmArc, topic: &str, msg: &[u8]) -> P2PRequestResult<()> {
     let uuid = match Uuid::from_str(topic) {
         Ok(u) => u,
-        Err(_) => return,
+        Err(e) => {
+            return MmError::err(P2PRequestError::DecodeError(e.to_string()));
```

Comment:
It's not logged further
`error!("Failed to get uuid from topic: {}", topic)` could be enough in this case

Reply:
Doesn't that lead to spam in logs. I mean one message  is here and one is there )

Reply:
Agree, I forgot to rollback these after logging it from caller.

Reply:
Those have different purposes as far as I see. Gives different idea with different structs. Caller logging the error directly, where inner only logs specific cases with more information in certain conditions.

Reply:
mmm, we should also return ok here https://github.com/KomodoPlatform/atomicDEX-API/blob/fe38f1862d625321aee11eb20a8f511f09b3ea69/mm2src/mm2_main/src/lp_swap.rs#L248-L252 since this means the message was `SwapStatus` message and it was decoded correctly. Will fix this.

Reply:
Done